### ISSUE TITLE
Add "xs" sizes for more compact UIs

### ISSUE THREAD

**Issue by mtancoigne** [inkline/inkline#47]
* **I'm submitting a:**

  - [ ] Bug Report
  - [x] Feature Request
  
* **What is the current behavior?**

There's only a "sm" size for UI elements (buttons/inputs,...) which is pretty big

* **What is the expected behavior?**

 It would be wonderful to have smaller UI elements to make compact UIs. We don't need to build interfaces for mobiles only (with finger accessible UI elements)

* **Please tell us about your environment:**

  - Version: current
  - Browser: n/a
  - Language: n/a

* **Other information** n/a

* **Personal note** InkLine is the best Vue UI framework I have tested; thank you all for your work !

**Comment by alexgrozav** [comment]
Hey! I'll work on some changes to allow users to add their own sizes inside the stylus config.

Thinking of something like:

~~~stylus
--component-sizes := (sm md lg)
~~~

Which you would be able to easily override to: 

~~~stylus
--component-sizes := (xs sm md lg xl)
~~~

**Comment by mtancoigne** [comment]
Thank you for your quick response; I think your implementation is really good :)

**Comment by alexgrozav** [comment]
I've implemented and refined this in 1.15.1.

Now you have a `--sizes` stylus variable which can be configured. By default, it will enable `(sm md lg)`, but you can now configure it to enable `(xs sm md lg xl)`, which will all be available out of the box.

~~~
--sizes = (xs sm md lg xl)
~~~

You can also configure `--size-multipliers` if you feel like it. See https://github.com/inkline/inkline/blob/master/src/css/config/_sizes.styl
